package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "@ember/optional-features": "0.7.0",
     "broccoli-asset-rev": "2.7.0",
+    "ember-angle-bracket-invocation-polyfill": "^2.0.2",
```

Comment:
We only need this one for tests, right? Otherwise this has to be moved from `devDependencies` to `dependencies`.

Reply:
I think that Ember apps lower than `3.10` would install `ember-angle-bracket-invocation-polyfill` into their own app dependencies.

That way there is no confusion about where angle bracket support comes from in their apps. 

This makes the most sense to me. What do you think?

Reply:
But would it be required if app uses curly component invocation syntax? I don't think so because angle bracket component invocation is not used in any of the component templates. But if I'm wrong we should document that requirement.

Reply:
I don't think `ember-angle-bracket-invocation-polyfill` is a requirement for consuming apps unless:

- The consuming app uses Ember < `3.10`
- AND invokes *any* component using angle-bracket syntax